### PR TITLE
rails-reformer project

### PR DESCRIPTION
**Pull request by MrObele**
In development stage, kindly review @talihomz

### CODE REVIEW

**Comment by MrObele** [comment]
@talihomz, project staging completed, here is the link to the review guidelines
https://docs.google.com/document/d/1YypxsoGs6ww2hL8Z11pMYG3v4nIhEPlWxc2bztAclCA/edit#heading=h.sbizsuk0weyb

**Comment by talihomz** [comment]
Hi @uzorjchibuzor 

I had a look at this. There aren't any instructions on the README as to how to run it, or what to expect when I do, so I took the following steps:
1. Switched to the staging branch
2. Installed all the gems.
3. Ran `rails db:create`
4. Ran `rails db:migrate`.

rails-reformer

Am I missing something here?

**Comment by uzorjchibuzor** [comment]
oops.. let me take a look at it

EDIT:
We did not change the root path. :see_no_evil: I have changed the ReadME. @talihomz

**Comment by talihomz** [comment]
Hi @uzorjchibuzor.

I finally saw something! But it isn't much, I think I need more info on the README on how to run this. Here is a screenshot of what i'm looking at.

screen shot 2019-01-25 at 12 36 22

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :users
+  root 'users#new'
+  get '/new', to: 'users#new'
+  get '/all', to: 'users#index'
+  get '/user/:id/edit', to: 'users#edit'
+  post '/users/create', to: 'users#create' 
 end
```
Comment (heyapricot) [inline config/routes.rb:8]: [The assignment says to use the `:only` symbol for creating the resources](https://guides.rubyonrails.org/routing.html#restricting-the-routes-created)

**Review by heyapricot — APPROVED** [review]
# It works 👍 
Left one comment. Take note and after that, you can move on 😄

**Review by uzorjchibuzor — APPROVED** [review]

**Pull request merged**